package.json

```diff
@@ -1,42 +1,46 @@
 {
```

Comment:
was this the result of `fixpack` or something?  if so, that needs to be a dep and part of the CI chain.